src/app/client/src/app/modules/dashboard/components/filter/filter.component.scss

```diff
@@ -0,0 +1,174 @@
+  .form-control {
+    min-width: 100%;
+    width: 100%;
+    height: calculateRem(32px);
+    font-size: calculateRem(12px);
+    border: calculateRem(1px) solid var(--gray-200);
+    border-radius: calculateRem(3px);
+    padding: calculateRem(8px);
+
+    &:focus {
+      border: calculateRem(1px) solid var(--rc-dddddd);
+      border-radius: calculateRem(3px);
+    }
+
+    &::placeholder {
+      color: var(--black);
+      opacity: 1;
+    }
+
+    &:-ms-input-placeholder {
+      color: var(--black);
+    }
+
+    &::-ms-input-placeholder {
+      color: var(--black);
+    }
+  }
+
+  .md-drppicker .ranges ul li {
+    margin-bottom: calculateRem(8px);
+  }
+
+  //   ngx-daterangepicker-material{
+  //     .md-drppicker.drops-down-right{
+  //         .ranges{
+
+  //         }
+  //     }
+  //   }
+}
+
+
+.sb-prominent-filter
+  .sb-prominent-filter-container
+  .sb-prominent-filter-field:last-child {
+  margin-left: 0;
+  margin-top: 0;
+  text-align: left;
+}
+
+.reset-filter-section {
+  position: absolute;
+  right: 0;
+  top: calculateRem(2px);
+  align-items: center;
+
+  .selection {
+    padding: calculateRem(8px);
+  }
+}
+
+.sb-filter-label {
+  border-bottom: calculateRem(1px) solid var(--rc-dddddd);
+  width: 100%;
+
+  .sb-label-value {
+    margin-bottom: calculateRem(4px);
+    margin-right: calculateRem(4px);
+  }
+}
+
+:host ::ng-deep {
+  .ui.toggle.checkbox input:checked ~ label,
+  .ui.checkbox input.hidden + label {
+    margin-bottom: 0px;
+  }
+
+  .ui.toggle.checkbox .box:before,
+  .ui.toggle.checkbox label:before {
+    background: rgba(var(--rc-rgba-black), 0.2);
+  }
+
+  .ui.toggle.checkbox .box:after,
+  .ui.toggle.checkbox label:after {
+    width: 1.5rem;
+    height: 1.2rem;
+    top: calculateRem(2px);
+  }
+
+  .ui.toggle.checkbox input ~ .box:after,
+  .ui.toggle.checkbox input ~ label:after {
+    left: 0.1rem;
+  }
+
+  .ui.toggle.checkbox input:checked ~ .box:after,
+  .ui.toggle.checkbox input:checked ~ label:after {
+    left: 1.8rem;
+  }
+
+  .ui.styled.accordion,
+  .ui.styled.accordion .accordion {
+    box-shadow: none;
+    border-bottom: calculateRem(1px) solid var(--rc-dddddd);
+  }
+
+  .graph-filters {
+    .ui.form {
+      min-height: calculateRem(36px);
+    }
+
+    .ui.styled.accordion .accordion .title,
+    .ui.styled.accordion .title {
+      padding-left: 0;
+      padding-right: 0;
+    }
+
+    .ui.styled.accordion .accordion .content,
+    .ui.styled.accordion .content {
+      padding: 0;
+    }
+
+    .ui.styled.accordion,
+    .ui.styled.accordion .accordion {
+      box-shadow: none;
+      border-bottom: calculateRem(1px) solid var(--rc-dddddd);
+    }
+  }
+}
+
+.sb-prominent-filter-field {
+  ::ng-deep {
+    .md-drppicker {
+      padding: calculateRem(16px) calculateRem(8px);
+
+      .ranges ul li {
+        margin-bottom: calculateRem(8px);
+      }
+
+      .btn {
+        background-color: var(--primary-400);
+        margin-right: calculateRem(16px);
```

Comment:
margin-right: calculateRem(16px);
 
 This line is repeated twice. Please remove the duplicate one

Reply:
cleared duplicated code